apps/condo/domains/notification/schema/SyncDeviceService.test.js

```diff
@@ -0,0 +1,419 @@
+// noinspection DuplicatedCode
+/**
+ * Generated by `createservice notification.SyncDeviceService --type mutations`
+ */
+const faker = require('faker')
+const sample = require('lodash/sample')
+
+const { makeClient, makeLoggedInClient } = require('@core/keystone/test.utils')
+
+const { syncDeviceByTestClient } = require('@condo/domains/notification/utils/testSchema')
+const { PUSH_TRANSPORT_TYPES } = require('@condo/domains/notification/constants/constants')
+
+const getRandomTokenData = (extraAttrs = {}) => {
+    const pushTransport = sample(PUSH_TRANSPORT_TYPES)
+    const data =  {
+        deviceId: faker.datatype.uuid(),
+        pushToken: faker.datatype.uuid(),
+        pushTransport,
+        meta: { pushTransport },
+        ...extraAttrs,
+    }
+
+    return data
+}
+
+/**
+ * Test cases for Service
```

Comment:
You have outlined the structure.
Please, organise your tests according to this structure.
Actually, there is no need for this extra comment.
A test structure should comment itself.
For example:

```
describe('anonymous/unauthorized', () => {
    describe('anonymous: deviceId + pushTransport', () => {
        it('registers device and pushTransport')
        it('ignores update operation')
    })
})

Reply:
fixed